client/src/bundles/bundle-cms.scss

```diff
@@ -1,7 +1,14 @@
-// Core variables
-//@import 'variables';
-
 @import '../styles/mfa-variables';
+@import '../framework/admin/scss/themes/_default';
+
+// Bootstrap reqs
+@import '~bootstrap/scss/functions';
+@import '~bootstrap/scss/variables';
```

Comment:
Should this not be before the `styles/mfa-variables`?

Reply:
It doesn't matter. They're essentially unrelated.

Reply:
Oh cool, good to know.
Do you think a comment might help people understand that?
I thought there would at least be a few colour variables that would get adjusted.

Reply:
I'm not really sure a comment is required. It's like `Whack bootstrap anywhere because this is the first time CMS has seen it. It doesn't matter where it's included`? It feels a bit like over-documentation at this point.